### DISCUSSION TITLE
The issue Dependency Dashboard is recreated each time the renovatebot is running

### DISCUSSION THREAD

**Discussion by ggrossetie** [renovatebot/renovate#14639] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### If you're self-hosting Renovate, tell us what version of Renovate you run.

v32.0.1

### Please select which platform you are using if self-hosting.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

I'm not really sure why but the "Dependency Dashboard" is recreated (instead of being updated) as you can see here: https://github.com/Mogztter/asciidoctor-kroki/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+dependency+dashboard+

![image](https://user-images.githubusercontent.com/333276/158073426-68098051-6db0-4962-b0af-5c2aa568cb1b.png)

![image](https://user-images.githubusercontent.com/333276/158073413-dee41d79-989d-4d09-a9e2-5fe427892a12.png)

```
DEBUG: Ensuring Dependency Dashboard (repository=Mogztter/asciidoctor-kroki)
DEBUG: ensureIssue(Dependency Dashboard) (repository=Mogztter/asciidoctor-kroki)
 INFO: Issue created (repository=Mogztter/asciidoctor-kroki)
```

I'm running the Renovate GitHub Action from a private repository:

```
Run renovatebot/github-action@v32.0.1
/usr/bin/docker run --env LOG_LEVEL --env RENOVATE_TOKEN=*** --env RENOVATE_CONFIG_FILE=/github-action/renovate.json --volume /home/runner/work/renovate-runner/renovate-runner/renovate.json:/github-action/renovate.json --volume /var/run/docker.sock:/var/run/docker.sock --volume /tmp:/tmp --user 1000:121 --rm renovate/renovate:32.0.1-slim
Unable to find image 'renovate/renovate:32.0.1-slim' locally
32.0.1-slim: Pulling from renovate/renovate
```

My configuration is:

```
{
  "branchPrefix": "renovate/",
  "includeForks": true,
  "username": "renovate-release",
  "gitAuthor": "Renovate Bot <bot@renovateapp.com>",
  "platform": "github",
  "repositories": [
    "asciidoctor/asciidoctor.js",
    "yuzutech/kroki",
    "yuzutech/kroki.io",
    "Mogztter/asciidoctor-web-pdf",
    "Mogztter/asciidoctor-kroki",
    "Mogztter/asciidoctor-opal-node-runtime",
    "Mogztter/unxhr",
    "yuzutech/annotations-action",
    "asciidoctor/asciidoctor-docbook.js",
    "asciidoctor/asciidoctor-cli.js",
    "asciidoctor/asciidoctor-browser-extension"
  ]
}
```

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment, marked as answer]
You incorrectly selected hosted app on GitHub.com when actually you're self hosting. 

This is also not a bug. 

Your gitAuthor is wrong, and needs to match your app's built-in email.

**Reply by rarkins** [reply]
Actually it might be the “username” field too. See https://docs.renovatebot.com/getting-started/running/ “Running as a GitHub app” section for details.

**Reply by ggrossetie** [reply]
Sorry about that and thanks for the lighting fast reply! 🥇 
I've updated both `username` and `gitAuthor` let's see how it goes.

**Reply by ggrossetie** [reply]
All good now 😌 

```
DEBUG: Ensuring Dependency Dashboard (repository=Mogztter/asciidoctor-kroki)
DEBUG: ensureIssue(Dependency Dashboard) (repository=Mogztter/asciidoctor-kroki)
DEBUG: Issue is open and up to date - nothing to do (repository=Mogztter/asciidoctor-kroki)
```

**Reply by viceice** [reply]
@rarkins @HonkingGoose maybe we should link to getting started page from platform auth config options?

**Reply by ggrossetie** [reply]
I guess we could also add a log to indicate that an issue named "Dependency Dashboard" was found but that the username/gitAuthor didn't match?

The documentation does not clearly indicate that the username/gitAuthor *must be* aligned with the Personal Access Token

> Regardless of platform, you need to select a user account for renovate to assume the identity of, and generate a Personal Access Token. It is recommended to be @renovate-bot if you are using a self-hosted server with free choice of usernames. It is also recommended that you configure config.gitAuthor with the same identity as your Renovate user, e.g. like "gitAuthor": "Renovate Bot <renovate@whitesourcesoftware.com>".

**Reply by rarkins** [reply]
No, because we do not find it at all. Renovate uses a GraphQL query to fetch only issues created by its own username, so if you specify the username wrong then it won't see any other dashboards at all.

**Reply by rarkins** [reply]
And regarding your second point, this problem doesn't apply when using PAT.

**Reply by HonkingGoose** [reply]
> maybe we should link to getting started page from platform auth config options?

@viceice Do you want to put a link to "Getting Started" on all of these pages?

- Overview page: https://docs.renovatebot.com/modules/platform/
  - Platform pages:
    - https://docs.renovatebot.com/modules/platform/azure/
    - https://docs.renovatebot.com/modules/platform/bitbucket/
    - https://docs.renovatebot.com/modules/platform/bitbucket-server/
    - https://docs.renovatebot.com/modules/platform/gitea/
    - https://docs.renovatebot.com/modules/platform/github/
    - https://docs.renovatebot.com/modules/platform/gitlab/

**Reply by rarkins** [reply]
Or maybe relocate some of the getting started into the platform readme's?

**Reply by HonkingGoose** [reply]
I like this "relocate getting started into the platform readme's" idea a lot! Want me to open a issue to track this idea?

**Reply by rarkins** [reply]
Yes, thanks. I heard feedback in the start that the getting started page was long/confusing so maybe if we break it up by platform then they have less to scroll through (but one more click to do..)

**Reply by HonkingGoose** [reply]
See issue #14650. 😉

**Reply by ggrossetie** [reply]
> And regarding your second point, this problem doesn't apply when using PAT.

I'm actually using a PAT and if the username/gitAuthor does not match it will recreate a "Dependency Dashboard" issue.

**Reply by viceice** [reply]
Yes, don't use `username` when using a `PAT` or make sure it matches the `PAT` user.

**Reply by rarkins** [reply]
Here's the docs for `username` currently:

![image](https://user-images.githubusercontent.com/6311784/158167573-1354d737-abe7-498b-84fc-81a172351f3f.png)

We should clarify better that the only two use cases are (a) Bitbucket, and (b) GitHub App with CLI. I think it's already fairly clear that it shouldn't be used with PAT.

**Reply by HonkingGoose** [reply]
> We should clarify better that the only two use cases are (a) Bitbucket, and (b) GitHub App with CLI. I think it's already fairly clear that it shouldn't be used with PAT.

See PR #14661. 😉
